scripts/Muon/GUI/ElementalAnalysis2/plotting_widget/EA_plotting_pane/EA_plot_data_pane_model.py

```diff
@@ -0,0 +1,112 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+from Muon.GUI.Common.plot_widget.base_pane.base_pane_model import BasePaneModel
+
+SPECTRA_INDICES = {"Delayed": 0, "Prompt": 1, "Total": 2}
```

Comment:
will we need this information elsewhere in the GUI?

Reply:
It is used in LoadElementalAnalysisData algorithm so i could import from there but the inverse is not present there

Reply:
are you likely to need them elsewhere in the GUI?

Reply:
I dont think they will be used elsewhere in the GUI

Reply:
ok